tests/runner.py

```diff
@@ -181,12 +195,17 @@ def hide_does_not_affect_capturing(self):
     class pseudo_terminals:
         def return_value_indicates_whether_pty_was_used(self):
             eq_(run("true").pty, False)
-            eq_(run("true", pty=True).pty, True)
+            if not WINDOWS:
+                eq_(run("true", pty=True).pty, True)
 
         def pty_defaults_to_off(self):
             eq_(run("true").pty, False)
 
         def complex_nesting_doesnt_break(self):
+            # Not sure how to make this work on Windows
+            if WINDOWS:
+                skip()
```

Comment:
Yeah, a decorator makes sense. Presumably tests/_utils.py would be a reasonable place to put it?